errors/page-data-collection-timeout.md

```diff
@@ -15,4 +15,5 @@ When restarted it will retry all uncompleted jobs, but if a job was unsuccessful
 
 ### Useful Links
 
-- [`getStaticPaths`](https://nextjs.org/docs/basic-features/data-fetching#getstaticpaths-static-generation)
```

Comment:
The caveat there is only within `/docs` - not `/errors` currently - unclear if we want to do that here too, maybe not as important.

Reply:
Yes, I saw that too. But fixed that just in case that was critical. 🙈